backend/app/models/visualization_config.py

```diff
@@ -17,6 +13,8 @@ class VisualizationConfigBase(BaseModel):
     job: Optional[EventListenerJobDB]
     client: Optional[str]
     viz_config_data: dict = {}
+    visualization: str
+    component_name: str
```

Comment:
Maybe change to `visualization_component_id` or something similar to be more specific?